### PR TITLE
Implementacion del card Home Page: Tabs de navegación

### PR DESCRIPTION
**Pull request by FloresFelix**
Summary
 implementación del card Home Page: Tabs de navegación

Screenshot

Trello card
https://trello.com/c/RIiYgK55

### CODE REVIEW

File: app/src/main/java/ar/com/wolox/android/example/ui/hompage/HomePageFragment.kt [hunk 1]
```diff
@@ -1,14 +1,53 @@
...
+
+    lateinit var tabLayout: TabLayout
+
     override fun layout() = R.layout.fragment_home_page
     override fun init() {
+        tabLayout = binding.tabLayout
+        binding.homeViewPager.adapter = SimpleFragmentPagerAdapter(childFragmentManager).apply {
+            addFragments(
+                newsFragment.get() to "news",
+                profileFragment to "profile"
```
Comment (ffasitella) [inline app/src/main/java/ar/com/wolox/android/example/ui/hompage/HomePageFragment.kt]: You can move "news" and "profile" to strings.xml.

File: app/src/main/java/ar/com/wolox/android/example/ui/hompage/HomePageFragment.kt [hunk 2]
```diff
@@ -1,14 +1,53 @@
...
+            )
+        }
+        tabLayout.setupWithViewPager(binding.homeViewPager)
+        tabLayout.getTabAt(0)?.setIcon(R.drawable.ic_news_icons)
+        tabLayout.getTabAt(1)?.setIcon(R.drawable.ic_profile_icon)
```
Comment (ffasitella) [inline app/src/main/java/ar/com/wolox/android/example/ui/hompage/HomePageFragment.kt]: `tabLayout.apply { } `

**Review by ffasitella — CHANGES_REQUESTED** [review]
Some small comments! 👍🏽

**Comment by FloresFelix** [comment]
Holaa!

Corregido las observaciones!!

**Review by ffasitella — APPROVED** [review]
Good job! Let's wait for the @jcruz-wolox's review!

**Pull request merged**